docs/metadata/Git/git-troubleshooting.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Git Troubleshooting
+id: git-troubleshooting
+description: How to troubleshoot permissions and network issues with Git
+tags: []
+---
+
+Sometimes there is an error when Prophecy tries to reach Git. This could be caused by your user's [authorization](#debugging-git-credentials), a [repository setting](#debugging-repository-setttings), or a [network](#debugging-network-issues) issue. Fear not! This guide outlines what to do if you see error messages such as:  
+`not authorized`  
```

Comment:
nit: to make it easier to read in this file, add empty spaces, so:

This guide outlines what to do if you see error messages such as:  

`not authorized`  
`git-upload-pack-not-permitted`  

or  

`git-recieve-pack-not-permitted`

Reply:
Or better yet, use an unordered list for these, so:

- not authorized
- git-upload-pack-not-permitted
- git-recieve-pack-not-permitted